### DISCUSSION TITLE
Add support for subtitles

### DISCUSSION THREAD

**Discussion by CodeCubeNeo** [N0-0NE-Dev/NoFasel#21] [Ideas]
Manual adding from file and if possible also from online providers. The best way would be like something from stremio (which is open source, so you can check out the code)

**Comment by abadawe-dev** [comment]
Yeah, this has been already suggested in #5, the problem is finding a website that is easy to scrape and has content with no subtitles. The websites I use right now has the subtitles baked into the video, so I cannot just switch the language.
